source/user-manual/reference/ossec-conf/localfile.rst

```diff
@@ -595,6 +597,96 @@ For example, we may want to read a Python Traceback output as one single log, re
+|             |                                         |   pcre2     |               |
++-------------+-----------------------------------------+-------------+---------------+
+
+For example, to ignore events related to configuration changes in the audit log:
+
+.. code-block:: xml
+
+  <localfile>
+      <log_format>audit</log_format>
+      <location>/var/log/audit/audit.log</location>
+      <ignore type="osregex">type=\.+_CHANGE</ignore>
+      <ignore type="osregex">type=CONFIG_\.+</ignore>
+  </localfile>
+
+.. note::
+  For formats that group multiple lines, the entire group will be counted as a single log to check if it should be ignored.
+
+.. note::
+  The ignore label is checked before the restrict label, so if you set identical expressions for both settings, no log will be processed.
+
+.. note::
+  On Windows, the ``eventchannel`` format already provides a way to ignore logs through queries, so this setting ``does not apply`` to this format.
+
+restrict
+^^^^^^^^^^^^^^^
+
+This allows you to configure a pcre2 expression to restrict specific log lines, avoiding to generate alerts for those logs that do not match. Multiple expressions can be configured by repeating the restrict label. The log will be processed only if all expressions match.
+
++--------------------+---------------------------------------------------------------+
+| **Default Value**  | n/a                                                           |
++--------------------+---------------------------------------------------------------+
+| **Allowed values** | Any `regex <regex.html#regex-os-regex-syntax>`_,              |
+|                    | `sregex <regex.html#sregex-os-match-syntax>`_ or              |
+|                    | `pcre2 <regex.html#pcre2-syntax>`_ expression.                |
++--------------------+---------------------------------------------------------------+
+
+The attribute type is optional.
+
++-------------+-----------------------------------------+-------------+---------------+
+| Attribute   |              Description                | Value range | Default value |
++=============+=========================================+=============+===============+
+| **type**    | Allows to set regular expression type   |   osregex   |     pcre2     |
+|             |                                         +-------------+               |
+|             |                                         |   osmatch   |               |
+|             |                                         +-------------+               |
+|             |                                         |   pcre2     |               |
++-------------+-----------------------------------------+-------------+---------------+
+
+For example, to restrict syslog events related to a particular user name:
+
+.. code-block:: xml
+
+  <localfile>
+      <log_format>syslog</log_format>
+      <location>/custom/file/path</location>
+      <restrict type="osregex">username_\.+</restrict>
```

Comment:
Change the type of this `restrict` to show that types can be different in the same `localfile`.